packages/polaris-viz/src/components/Annotations/Annotations.tsx

```diff
@@ -52,26 +52,46 @@ export function Annotations({
 
     const annotations = Object.keys(annotationsLookupTable)
       .map((key) => {
-        const annotation = annotationsLookupTable[key];
+        const annotation: Annotation | null = annotationsLookupTable[key];
 
         if (annotation == null || annotation.axis === 'y') {
           return null;
         }
 
-        const formattedKey = labelFormatter(key);
```

Comment:
Does this mean the `startAxisLabel` passed in will need to be formatted by the consumer? Would a boolean prop to toggle formatting of the key (as @philschoefer suggested) be easier? That way the consumer wouldn't have to worry about running this `startAxisLabel` through their label formatter before, they could just toggle it and polaris viz would take care of the formatting